package main.exceptions;

public class UserNameAlreadyExistsException extends DataException{
    /**
     * Wird geworfen wenn der gew√§hlte Name bereits vergeben ist
     */
    public UserNameAlreadyExistsException() {
    }

}
